config/west.yml

```
@@ -2,6 +2,8 @@ manifest:
   remotes:
     - name: zmkfirmware
       url-base: https://github.com/zmkfirmware
+  # Additional modules containing boards/shields/custom code can be listed here as well
+  # See https://docs.zephyrproject.org/latest/develop/west/manifest.html#projects
```

```suggestion
    # Additional modules containing boards/shields/custom code can be listed here as well
    # See https://docs.zephyrproject.org/3.2.0/develop/west/manifest.html#projects
```
Some indentation to match the list, and how about version pinning?